notebooks/fmrisim/fmrisim_multivariate_example.ipynb

```diff
@@ -111,7 +111,7 @@
    "source": [
     "*1.2 Load participant data*<a id=\"load_ppt\"></a>\n",
     "\n",
-    "Any 4 dimensional fMRI data that is readible by nibabel can be used as input to this pipeline. For this example, data is taken from the open access repository DataSpace: http://arks.princeton.edu/ark:/88435/dsp01dn39x4181. This file is unzipped and placed in the home directory with the name Corr_MVPA "
+    "Any 4 dimensional fMRI data that is readible by nibabel can be used as input to this pipeline. For this example, data is taken from the open access repository DataSpace: http://arks.princeton.edu/ark:/88435/dsp01dn39x4181. This file is unzipped and placed same directory as this notebook with the name Corr_MVPA "
```

Comment:
@CameronTEllis please note a minor update to the data directory to make it compatible for automated testing.